bowling/BowlingAlley.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <filesystem>
+#include <memory>
+#include <vector>
+
+namespace fs = std::filesystem;
+class Game;
+class BowlingAlley {
+public:
+    BowlingAlley() = default;
```

Comment:
why not delete?

Reply:
I don't understand why would it be a problem? I thought that it's always good to make a default constructor, even though it would only set members to default values. For example you can create an array of `BowlingAlley` thanks to default constructor

Reply:
Not always. Example - you have a class Person that can take a name and PESEL. There should be no possibility to create a person without name and PESEL in a database. The same here - we do not want to have such "empty" BowlngAlley, because it always must be created based on existing file.